packages/cli/src/run-test.ts

```diff
@@ -8,9 +8,24 @@ import * as query from './query';
 import { StreamingFormatter } from './format-helpers';
 
 export function* runTest(config: ProjectOptions, formatter: StreamingFormatter): Operation<void> {
-  let client: Client = yield Client.create(`ws://localhost:${config.port}`);
+
+  let client: Client;
+  let uri = `ws://localhost:${config.port}`;
+
+  try {
+    client = yield Client.create(uri);
+  } catch (e) {
+    if (e.message.includes('websocket server closed connection unexpectedly')) {
```

Comment:
We could also modify the client to throw a more informative error, so that we could ask `e.name === 'NoServerError` or something. That way it would be more robust for future users of `Client`. Not sure how that would work with TypeScript. Thoughts?

Reply:
Done